content/source/assets/javascripts/application.js

```diff
@@ -3,3 +3,18 @@
 
 //= require hashicorp/mega-nav
 //= require hashicorp/sidebar
+
+document.addEventListener("turbolinks:load", function() {
```

Comment:
I think we also need to add an event listener for "body:load" too, since this will only activate on nav to the page, but I don't think it will activate on direct visit.

Reply:
Nah, [it fires on both](https://github.com/turbolinks/turbolinks#running-javascript-when-a-page-loads) (I tested to make sure). Luckily they took the 99%-of-the-time use case into account when designing their event API. 😅